al2/rust/hello/{{ cookiecutter.project_slug }}/README.md

```diff
@@ -33,14 +33,14 @@ To deploy the application, you need the folllowing tools:
 
 * SAM CLI - [Install the SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
 * Docker - [Install Docker community edition](https://hub.docker.com/search/?type=edition&offering=community)
-* [Rust](https://www.rust-lang.org/) version 1.56.0 or newer
-* [cargo-lambda](https://github.com/cargo-lambda/cargo-lambda#installation) for cross-compilation
+* [Rust](https://www.rust-lang.org/) version 1.64.0 or newer
+* [cargo-lambda](https://github.com/cargo-lambda/cargo-lambda) for cross-compilation
 
 To build and deploy your application for the first time, run the following in your shell:
 
 ```bash
 sam build
-sam deploy --guided
+sam deploy
 ```
 
 The first command will build the source of your application. The second command will package and deploy your application to AWS, with a series of prompts:
```

Comment:
Same comment as above re: mentioning `-g` or `--guided` so that the rest of this makes sense